src/component/header/ZeroFillingOptionsPanel.tsx

```diff
@@ -83,17 +77,9 @@ function ZeroFillingOptionsInnerPanel(props: { size: number }) {
           onChange={handleChangeSizeHandler}
         />
       </Label>
-      <Label
-        title="Live preview "
-        htmlFor="livePreview"
-        style={headerLabelStyle}
-      >
-        <CheckBox
-          name="livePreview"
-          defaultChecked
-          onChange={checkChangeHandler}
-        />
-      </Label>
+      <Field label="Live preview" name="livePreview">
+        <Checkbox checked={livePreview} onChange={setLivePreview} />
+      </Field>
```

Comment:
Should not use `Field`, but `label` prop on the chekbox (it will put the text on the right, but I think it's better). Also I think you can remove the "name"

Reply:
Okay, i will fixe it. It just that the label looked better before the checkbox